### DISCUSSION TITLE
Arc Overhang Updates!

### DISCUSSION THREAD

**Discussion by stmcculloch** [stmcculloch/arc-overhang#16] [Announcements]
Over the past couple months, a small group of helpful community members have taken arc overhangs to the next level! We now have the tools available to actually use arc overhangs in real 3D prints!

## Pleccer - Arc Overhangs in a slicer!

@rvmn has implemented arc overhangs in their slicer "Pleccer"! [Follow progress here](https://github.com/rvmn/SuperPleccer/discussions)

Pleccer is a fork of PrusaSlicer, and in addition to arc overhangs, it also includes: 
- Improved color picker (thanks to [colorgrab](https://github.com/nielssp/colorgrab))
- Mixing color calibration setting (requires a (diy) color measurer)
- Mixing color g-code generation based on calibrated data and the color you pick
- Enhanced smart retraction and gradient change calibration and automatic model updating for purge-free printing (no prime tower/purge)
- Includes Arc overhang infill https://github.com/stmcculloch/arc-overhang/issues/10 for support-free printing with 'spiral'-mode for pedestal-connected overhangs
- Includes smart island-only tree support setting for use with arc infill overhangs to have only the really required supports printed
- Includes Lightning infill for faster printing
- Includes Arachne slicing
- It has also all the current Bambu Studio's features

## Post-processing script - Add arc overhangs to existing gcode files!

@nicolai-wachenschwan has created a post-processing script for PrusaSlicer that can be used to generate arc overhangs! It's all in Python so it is quite easy to modify and add your own changes. [See progress here](https://github.com/nicolai-wachenschwan/arc-overhang-prusaslicer-integration)

This python script works quite well, however it is not incorporated directly into the slicer and rather works directly from the final gcode. I believe it is best used as a testing platform to quickly try out new algorithms and quickly iterate since python is a lot easier to work with.

We hope that others can take Nicolai's platform as a starting point to try their own improvments, as there are still [many good ideas for arc overhangs that we haven't tested yet](https://github.com/stmcculloch/arc-overhang/discussions/14)

## Call to action

If you want to see arc overhangs find their way into mainstream slicers, (Cura, PrusaSlicer, SuperSlicer, Bambu Studio, etc) we need to prove that they are effective and reliable! Therefore if you are willing to help test or work on the algorithm design, please reach out! Check here for some ideas to get started: https://github.com/stmcculloch/arc-overhang/discussions/14

**Comment by Davidborzaga** [comment]
Ho provato a caricare questo script su PrusaSlicer ma quando attivo il post processore mi da errore 193 non capisco come mai
